core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -107,7 +108,6 @@
     private static final String PREF_RIGHT_VOLUME = "prefRightVolume";
 
     // Experimental
-    public static final String PREF_SONIC = "prefSonic";
```

Comment:
I realize this has already been merged, but I just wanted to say that the sonic player test still depends on this.
https://github.com/AntennaPod/AntennaPod/blob/develop/app/src/androidTest/java/de/test/antennapod/ui/PlaybackSonicTest.java#L62

That being said, I just started looking into this codebase this evening and as of right now all tests are still failing for me, so I'm not sure whether I should make a PR for that. 😅

Reply:
I'm currently working on fixing the tests in #2870 ;)